_std/defines/lightcycles.dm

```diff
@@ -0,0 +1,39 @@
+// m = 127 + cos(BUILD_TIME_HOUR / 12 * (pi * 2) * 127
```

Comment:
find a better name imo, this makes me think of Tron and not day/night-cycles

Reply:
done! good catch, i knew it was a bit funky.